app/views/layouts/application.html.erb

```diff
@@ -8,12 +8,14 @@
 </head>
 <body>
   <div id="wrapper">
-    <ul class="nav">
-      <li><%= link_to_unless current_page?(root_path), "Govspeak Preview", root_path %></li>
-      <li><%= link_to_unless current_page?(guide_index_path), "Govspeak Guide", guide_index_path %></li>
-      <li><%= link_to_unless current_page?(convert_index_path), "Convert Google Doc to Govspeak", convert_index_path %></li>
-    </ul>
-    <%= yield %>
+    <%= render "govuk_publishing_components/components/layout_header", {
+      environment: "example",
```

Comment:
It's coming up with example in the purple box; does removing it work?

```suggestion
      environment: "",
```

Reply:
@injms That's intentional - I guess it flags that this is something which GOVUK have built but isn't a live page on GOV.UK, seeing as this is available to the public. This may be even more important if we move this from heroku to a service domain.

We use the same thing on our govuk_publishing_components component guide: https://components.publishing.service.gov.uk/component-guide

Reply:
I see - that makes sense. Thanks for clarifying :)